android/app/src/main/java/com/onair/hearit/data/datasource/MemberRemoteDataSourceImpl.kt

```
@@ -2,18 +2,35 @@ package com.onair.hearit.data.datasource
-            val response = memberService.getUserInfo()
+            val response = memberService.getUserInfo(getAuthHeader())
             when (response.code()) {
-                200 -> response.body() ?: throw IllegalStateException("응답 바디가 null입니다.")
-                401 -> throw UserNotRegisteredException("회원가입되지 않은 사용자입니다.")
+                200 ->
+                    response.body() ?: throw IllegalStateException(
+                        ERROR_RESPONSE_BODY_NULL_MESSAGE,
+                    )
+
+                401 -> throw UserNotRegisteredException(ERROR_SIGN_IN_MESSAGE)
                 else -> throw Exception("API 호출 실패: ${response.code()} ${response.message()}")
```

detekt (1.23.8)

[warning] Exception is a too generic Exception. Prefer throwing specific exceptions that indicate a specific error case.

(detekt.exceptions.TooGenericExceptionThrown)